### DISCUSSION TITLE
Library for Axis Joystick

### DISCUSSION THREAD

**Discussion by MichaelKov** [AxisCommunications/discussions#225] [Axis Windows SDKs] [answered]
Hallo everybody.
i have a question. I try to write my own Joystick program. I have dowlload Joystick SDK with 3 Exampls. Anfortunatly , in debag Mpdus , comes ap error message: "The type or namespace name 'AxisJoystickModuleLib' could not be found (are you missing a using directive or an assembly reference?)". Does anyone know how can i fix that problem? Maybe someone know where i can download AxisJoystickModuleLib? 

Thank for answering

Michael

**Comment by shadysp** [comment]
Could you please check that the issue is not due to .NET Framework incompatibility\mismatch version or profiling.

**Reply by ghost** [reply, marked as answer]
The AxisJoystickModuleLib is available after registering the “AxisJoystickModuleX64.dll” and this file is included in the SDK and registered during setup. Try reinstalling the SDK and that the latest SDK version is used. 

Verify that the correct project (x64/x86) is selected when building. See if AxisJoystickModuleLib is shown in references in Visual Studio. 
If not or warning is shown. Try right click and add reference/COM and find “AXISJoystickModeule……..” 
If you cannot find it open a windows “cmd” windows as administrator go to the bin folder containing “AxisJoystickModuleX64.dll” and type “regsvr32  AxisJoystickModuleX64.dll”.
Then try adding reference again.

If this does not help what Visual Studio version is used ?
What samples are used C#/CPP ?
